packages/web/src/shared/Footer.tsx

```diff
@@ -64,35 +63,27 @@ const Social = React.memo(function _Social() {
           <MediumLogo color={colors.dark} height={height} />
         </View>
         <View style={styles.socialIcon}>
-          <Link href={CeloLinks.gitHub}>
-            <a>
-              <Octocat color={colors.dark} size={height} />
-            </a>
-          </Link>
+          <a target="_blank" rel="noopener" href={CeloLinks.gitHub}>
```

Comment:
this was throwing warnings for a while now, glad to have finally found out where they came from